port/cpl_float.h

```diff
@@ -693,4 +693,6 @@ template <typename T> constexpr int CPLIsFinite(T x)
 
 #endif  // #ifdef __cplusplus
 
+double CPLGreatestCommonDivisor(double x, double y);
```

Comment:
if a C++ unit test is added:
```suggestion
double CPL_DLL CPLGreatestCommonDivisor(double x, double y);
```